packages/flutter_hooks/lib/src/draggable_scrollable_controller.dart

```diff
@@ -0,0 +1,34 @@
+part of 'hooks.dart';
+
+/// Creates a [DraggableScrollableController] that will be disposed automatically.
+///
+/// See also:
+/// - [DraggableScrollableController]
+DraggableScrollableController useDraggableScrollableController(
+    {List<Object?>? keys}) {
```

Comment:
A little trailling comma would be great :)
```suggestion
    {List<Object?>? keys,}) {
```